### PR TITLE
Literals

### LINKED ISSUE #12: write literals chapter

### PR DESCRIPTION
**Pull request by F1rst-Unicorn**
Completed literals chapter
Closes #12

### CODE REVIEW

File: spec/literals.tex [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+\begin{itemize}
+	\item \texttt{0}
+	\item \texttt{42}
+	\item \texttt{-1}
+	\item \texttt{1009}
+\end{itemize}
+
+
+\subsection{Hexadecimal numbers}
+To represent numbers in the hexadecimal system Lizard uses the same syntax as Java, C and alike. The number is preceeded by the prefix \texttt{0x} after which at least one hexadecimal digit (\texttt{0123456789abcdefABCDEF}) follows. Note that you cannot use the minus sign here.
```
Comment (ntruessel) [inline spec/literals.tex]: Mention two's complement encoding for negative numbers?

Comment (F1rst-Unicorn) [inline spec/literals.tex]: Is already mentioned in the types chapter, and these literals are of that type. Isn't that redundant?

Comment (ntruessel) [inline spec/literals.tex]: I think it would not harm. but we can leave it as is as well

Comment (F1rst-Unicorn) [inline spec/literals.tex]: Added, with reference (latex rules :P)

Comment (ntruessel) [inline spec/literals.tex]: Oh, that seems like a misunderstanding. I thought you'd write it to the Hexadecimal section, as it only matters there (big endian, two's complement)

Comment (F1rst-Unicorn) [inline spec/literals.tex]: but why do I care about two's complement when setting bits? and I think we can define it endian independent, just stating that the MSB is to the left

Comment (ntruessel) [inline spec/literals.tex]: In case someone wants to set bits, but know the value as well, he has to know the exact encoding.

Comment (F1rst-Unicorn) [inline spec/literals.tex]: But endianness only matters for the encoding Register <--> Memory, and in Lizard we don't allow memory tampering. If the value is in use it is in the exactly same format as it has been written in the literal

Comment (ntruessel) [inline spec/literals.tex]: But if we define the hex encoding to be little endian, 0x00112233 becomes, what in C would be 0x33221100. Similarly, if we chose to use one's complement 0xffffffff is 0 as opposed in two's complement.

**Pull request merged**